lib/scout_apm/layer_converters/trace_converter.rb

```diff
@@ -117,7 +117,9 @@ def create_spans(layer, parent_id = nil)
           tags)
 
         layer.children.each do |child|
-          unless over_span_limit?(result)
+          # Don't create spans from limited layers. These don't have start/stop times and our processing can't
+          # handle these yet.
+          unless over_span_limit?(result) or child.is_a?(LimitedLayer)
```

Comment:
Why use `or` instead of `||`

Reply:
Yep - good catch! Not control flow. Updated!